content/tracks/Middleware.md

```diff
@@ -1,3 +1,15 @@
 ---
 title: Middleware
----
\ No newline at end of file
+---
+
+Middleware is the most widely existing infrastructure based on application system, which is used to process with various non-functional requirements.
+They include web middleware, message middleware, database middleware, and someone to achieve specific requirements.
```

Comment:
Is it Ok to remove the messaging middleware? Seems we have a track for Message, it may be confusing for speaker to choose.